roles/agent/README.md

```diff
@@ -4,11 +4,33 @@ Install and configure a [Beszel](https://github.com/henrygd/beszel) binary agent
 
 ## Role Variables
 
+### Authentication Methods
+
+Beszel supports two authentication methods:
+
+#### Method 1: Individual System Authentication (Traditional)
+
 ```yaml
 agent_public_key: ""
+agent_token: ""
+# Example
+agent_public_key: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIJxK7vQ9rL2mN8pYfGhU3zWcE5tRoS6iD1bV4nM9qX8A"
+agent_token: "944bb14b-25d7-47f8-bf79-ca9c048a3370"
+```
+
+Individual system authentication using both public key and system token. Requires manual system creation in the hub where you provide the public key and receive the corresponding token.
```

Comment:
Could you please add to the wording that a user could also use the `community.beszel.system` module too?

> Individual system authentication using a public key and system token. Requires manual system registration in Beszel hub where you provide the public key and receive the corresponding token. Alternatively, the `community.beszel.system` module can be used to the register the system.